backend/database_handler/accounts_manager.py

```diff
@@ -1,87 +1,89 @@
 # consensus/services/transactions_db_service.py
 
-import json
-from enum import Enum
 from eth_account import Account
 from eth_account._utils.validation import is_valid_address
 
-from backend.database_handler.db_client import DBClient
+from .models import CurrentState
 from backend.database_handler.errors import AccountNotFoundError
 from backend.database_handler.transactions_processor import TransactionsProcessor
 
+from sqlalchemy.orm import Session
+
 
 class AccountsManager:
-    def __init__(
-        self, db_client: DBClient, transactions_processor: TransactionsProcessor
-    ):
-        self.db_client = db_client
+    def __init__(self, session: Session, transactions_processor: TransactionsProcessor):
+        self.session = session
         self.transactions_processor = transactions_processor
-        self.db_accounts_table = "current_state"
 
-    def _parse_account_data(self, account_data: dict) -> dict:
+    def _parse_account_data(self, account_data: CurrentState) -> dict:
```

Comment:
@AgustinRamiroDiaz do we still need this? why can't we just return the same CurrentState object?

Reply:
talked async: we'll not use the database models as our domain objects. At this layer, we need to convert to our domain
currently our domain for accounts is `dict`. We'll change that in the future, is out of scope for this PR